app/models/my_module.rb

```diff
@@ -248,14 +248,14 @@ def number_of_samples
   end
 
   def is_overdue?(datetime = DateTime.current)
-    due_date.present? and datetime.utc > due_date.utc
+    due_date.present? && datetime.utc > due_date.end_of_day
```

Comment:
I'm not completely sure here, so this is more of a question: shouldn't we compare 2 UTC dates here? Or is this on purpose because we're considering some local date?

Reply:
No, you're completely right, I'm being sloppy here, just did a quick search and replace.

Reply:
My apologizes for taking your time.